Automattic/wp-calypso
# Pattern Library: Replace SVGs with PNGs in `PatternsGetStarted`

Fixes https://github.com/Automattic/dotcom-forge/issues/6141

## Proposed Changes

The current images in `PatternsGetStarted` have been exported as SVGs from Figma (BnTfb6hzKcRTb2LBtuPwKe-fi-3%3A950), but they actually contain bitmaps. I've exported the images again from the highest-resolution versions I could find in Figma, resized and compressed them, and applied an unsharp mask to `understand-page-layouts.png` and `use-block-patterns.png` since those images were quite unsharp in their originals.

## Testing Instructions



1. Navigate to `/patterns`
2. Scroll down to the `All about patterns` section
3. Ensure that the images look good and uniform
